resources/views/projects/details.blade.php

```diff
@@ -284,15 +316,20 @@
 <script>
 	/* Semantic UI setup */
 	$(".menu .item").tab();
-	$("#new-task-modal").modal({ transition: "fade up" });
-	$("#supplier-to-add-modal").modal({ transition: "fade up" });
-	$("#supplier-to-add-error-modal").modal({ transition: "fade up" });
+	$('#progress').progress();
 
+	@if($project->isStakeholder(Auth::id()))
+		$(".modal").modal({ transition: "fade up" });
 
+		/* Task details modal */
+		$(".task-title").click(showTaskDetails);
 
-	$(document).ready(function() {
-		$(".ui.fluid.search.dropdown").dropdown();
-		$('#progress').progress();
+		function showTaskDetails() {
```

Comment:
Si no estás usando esto anywhere else, you should make this an anonymous function y pasarselo al click handler